tests/gtests/sw_testhelpers.cc

```diff
@@ -80,7 +74,8 @@ void Reset_SOILWAT2_after_UnitTest(void) {
 
 
 
-void create_test_soillayers(unsigned int nlayers, char *InFiles[]) {
+void create_test_soillayers(unsigned int nlayers, char *InFiles[],

```

Comment:
`create_test_soillayers()` also utilizes "LogInfo" and needs this is an argument